knapsack.py

```diff
@@ -0,0 +1,28 @@
+# Time Complexity : O(M*N) where M is no of weight, N is capacity
+# Space Complexity : O(M*N) where M is no of weight, N is capacity //?
+# Did this code successfully run on Leetcode : not in Leetcode, executed in PyCharm
+# Any problem you faced while coding this : the video was very helpful
```

Comment:
Great that the video helped you understand the problem. Good job using resources